Unaccountable-Magpie/BangazonSite
# product type details

### Purpose:
show products that are of a certain type


### How it fits in context of the project:
(Link to ticket: # )

#2 
### Specific Feature affected:
product types details view


### How to test (Be thorough!):

1.git fetch --all

git checkout jr-bang

open nugget console and run dotnet test

you should get 3 passes and 0 fails when it runs

press execute and go to the webpage and click on Product Types in the nav bar

then click on clothing and all the clothing in our store will show up with prices and quantities
